modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -564,6 +565,10 @@ export default class I3SConverter {
         this.options.sevenZipExe
       );
 
+      const hashTable = await composeHashFile(new FileHandleFile(slpkFileName));
+
+      await addOneFile(slpkFileName, hashTable, '@specialIndexFileHASH128@');
+
```

Comment:
Too much empty space on my sense

```suggestion
      const hashTable = await composeHashFile(new FileHandleFile(slpkFileName));
      await addOneFile(slpkFileName, hashTable, '@specialIndexFileHASH128@');

```